components/brave_wallet_ui/panel/style.ts

```diff
@@ -4,16 +4,15 @@
 // you can obtain one at https://mozilla.org/MPL/2.0/.
 import styled from 'styled-components'
 
-interface StyleProps {
+export const PanelWrapper = styled.div<{
   isLonger: boolean
-}
-
-export const PanelWrapper = styled.div<StyleProps>`
+  isV2?: boolean
```

Comment:
How about defining width/height props to override the v1 style instead?

Reply:
Good idea, added width/height props!